arches/app/templates/login.htm

```diff
@@ -75,7 +75,7 @@ <h4>{% trans "Login failed" %}</h4>
 			</div>
 
 			<div class="panel-footer">
-				Learn more about <a href="archesproject.org">{% trans "Arches" %}</a>
+				{% trans "Learn more about" %} <a href="archesproject.org">{% trans "Arches" %}</a>
```

Comment:
this whole line including tags should be wrapped in a "trans" tag, this way the translator can put the emphasis in the correct context
```{% trans "Learn more about <a href='archesproject.org'>Arches</a>" %}```